examples/branch-protection/main.tf

```diff
@@ -0,0 +1,51 @@
+data "github_team" "admin" {
+  slug = "AdminTestTeam"
+}
+
+data "github_team" "maintain" {
+  slug = "MaintainTestTeam"

```

Comment:
Do these slugs exist? We could add them as empty teams to test the build.

Reply:
I have added some teams for test purposes.